roles/users/templates/sudoers.d.j2

```diff
@@ -0,0 +1,5 @@
+# {{ ansible_managed }}
+
+{% for service in web_sudoers %}
+{{ web_user }} ALL=(ROOT) NOPASSWD: {{ service }}
+{% endfor %}
```

Comment:
The deploy role's "Reload php-fpm" task only works for me if run as `root` instead of `ROOT`.
@mAAdhaTTah This task was hanging for you too. Does this change resolve the issue?
```diff
- {{ web_user }} ALL=(ROOT) NOPASSWD: {{ service }}
+ {{ web_user }} ALL=(root) NOPASSWD: {{ service }}
```